### ISSUE TITLE
Windows could not set the offline locale information and script crashing

### ISSUE THREAD

**Issue by ayoGrox** [memstechtips/WIMUtil#11]
https://github.com/user-attachments/assets/e8e6431d-b617-49bc-83e4-483fc904c120

Hello!

I had been trying to get past this error during windows install

Idk what is causing this but I have Ethernet plugged in and I managed to get it installed once before the boot gave up

And the script is crashing when it's copying iso files. I had tried a different computer and still the same thing... This might only affect windows 11 or not idk

So if anyone got a way around it thank you so much for your help!

Also here's the photo attached

**Comment by ayoGrox** [comment]
holy shit still no response

**Comment by ayoGrox** [comment]
You do know that the issue is still happening

**Comment by memstechtips** [comment]
Hey, can you give some more info:

1. What ISO file are you using?
2. What answer file are you using? UnattendedWinstall or your own?

Neither WIMUtil nor UnattendedWinstall changes anything when it comes to offline locale info, so I'm not sure where this issue is coming from.

**Comment by ayoGrox** [comment]
Hello!

Im using the windows 10 ISO (From official source)

And no im not using my own answer file im using the one from your GitHub

And i also dont know what it going on Cuz I tested with imported drivers the issue still happens

**Comment by memstechtips** [comment]
Yeah that is really weird because I haven't been able to recreate that error yet.

**Comment by ayoGrox** [comment]
Idk must be Something wrong with my pc then

I hope it isnt

**Comment by ayoGrox** [comment]
Also i had been getting an error that The installer Could not find some type of files idk what the error was but i can try and send a screenshot of it

**Comment by ayoGrox** [comment]
https://github.com/user-attachments/assets/112feda5-ed1c-4b03-b5e7-53fa463641ac

Here's the screenshot attached

**Comment by ayoGrox** [comment]
Aight so the problem persists on windows 10 Pro

windows 11 Pro works as normal
So im gonna wait on a next update

**Comment by almargb** [comment]
This problem happened to me. The answer file must be added correctly in a Windows image in the boot folder No. 1 and No. 2 and this way the problem will be solved.

**Comment by vinicolares** [comment]
> This problem happened to me. The answer file must be added correctly in a Windows image in the boot folder No. 1 and No. 2 and this way the problem will be solved.
> 
> The same answer file is added in both folders

Could you explain what these two “boot” folders are, with pictures if possible? -_-'

**Comment by almargb** [comment]
This method was told to me by a friend.
After installing the Windows image on the flash drive using rufus from the flash drive, open the sources folder, then the boot folder, then paste the answer file in folder number 1 and folder 2, then install Windows 11.

**Comment by memstechtips** [comment]
This project is now archived and has been superseded by [Winhance](https://github.com/memstechtips/Winhance).
